components/sup/src/event/error.rs

```diff
@@ -13,7 +14,8 @@ pub enum Error {
     ConnectNatsServer,
     HabitatCore(habitat_core::Error),
     NativeTls(native_tls::Error),
-    Rants(RantsError),
+    Nats(NatsError),
```

Comment:
I'd change `Nats` to `IOError` since there is no guarantee than an io::error is going to be a Nats related error.